src/node_crypto.cc

```diff
@@ -3786,10 +3786,8 @@ void CipherBase::SetAuthTag(const FunctionCallbackInfo<Value>& args) {
   // Restrict GCM tag lengths according to NIST 800-38d, page 9.
   unsigned int tag_len = Buffer::Length(args[0]);
   if (tag_len > 16 || (tag_len < 12 && tag_len != 8 && tag_len != 4)) {
-    ProcessEmitWarning(cipher->env(),
-        "Permitting authentication tag lengths of %u bytes is deprecated. "
-        "Valid GCM tag lengths are 4, 8, 12, 13, 14, 15, 16.",
-        tag_len);
+    Environment* env = Environment::GetCurrent(args);
+    return env->ThrowError("Invalid GCM authentication tag length");
```

Comment:
I would prefer keeping the invalid `tag_len` in the message for debuggability, but it's not blocking.

Reply:
Thank you, done! :)